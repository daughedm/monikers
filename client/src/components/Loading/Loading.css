.loading {
  height: 80vh;
  display: flex;
  flex-direction: column;
  justify-content: center;

}

.loading-gif {
  margin: auto;
  display: block;
}