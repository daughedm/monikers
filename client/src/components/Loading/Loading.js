import React from 'react';
import logolockup from '../../assets/Monikers_logo_lockup-02.svg';
import loadingGif from '../../assets/loading.svg';
import './Loading.css';

const Loading = () => {
  return (
    <div className="loading">
      <img className="logo-lockup" src={logolockup} alt="logolockup" />
      <img className="loading-gif" src={loadingGif} alt="loadingGif" />
    </div>
  );
};

export default Loading;
