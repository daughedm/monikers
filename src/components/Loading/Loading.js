import React, { Component } from 'react';
import logolockup from '../../assets/Monikers_logo_lockup-02.svg';
import loadingGif from '../../assets/loading.svg';
import './Loading.css';

export class Loading extends Component {
  componentDidMount() {}

  render() {
    return (
      <div className="loading">
        <img className="logo-lockup" src={logolockup} />
        <img className="loading-gif" src={loadingGif} />
      </div>
    );
  }
}

Loading.propTypes = {};

export default Loading;
