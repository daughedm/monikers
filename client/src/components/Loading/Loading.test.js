/* eslint-disable max-len */

import React from 'react';
import { shallow } from 'enzyme';
import Loading from './loading';

describe('loading', () => {
  let loading;

  beforeEach(() => {
    loading = shallow(<Loading />);
  });

  it('matches the snapshot', () => {
    expect(loading).toMatchSnapshot();
  });
});
