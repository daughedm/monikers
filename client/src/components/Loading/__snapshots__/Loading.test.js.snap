// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="logolockup"
          className="logo-lockup"
          src="Monikers_logo_lockup-02.svg"
/>,
        <img
          alt="loadingGif"
          className="loading-gif"
          src="loading.svg"
/>,
      ],
      "className": "loading",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "logolockup",
          "className": "logo-lockup",
          "src": "Monikers_logo_lockup-02.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "loadingGif",
          "className": "loading-gif",
          "src": "loading.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="logolockup"
            className="logo-lockup"
            src="Monikers_logo_lockup-02.svg"
/>,
          <img
            alt="loadingGif"
            className="loading-gif"
            src="loading.svg"
/>,
        ],
        "className": "loading",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "logolockup",
            "className": "logo-lockup",
            "src": "Monikers_logo_lockup-02.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "loadingGif",
            "className": "loading-gif",
            "src": "loading.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
